### PR TITLE
added apis for get accommodation, refactoring and upgrade to .net5

### PR DESCRIPTION
**Pull request by hasmikharutyunyan**

### CODE REVIEW

**Comment by kirillta** [comment]
Should't we rename the project to `AccommodationMapper`, what do you think?

File: HappyTravel.PropertyManagement.Api/Services/Workers/IAccommodationsDataMerger.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+using HappyTravel.EdoContracts.Accommodations;
+using HappyTravel.PropertyManagement.Data.Models.Accommodations;
+
+namespace HappyTravel.PropertyManagement.Api.Services.Workers
+{
+    public interface IAccommodationsDataMerger
+    {
+        Task MergeAccommodationsData(CancellationToken cancellationToken);
+
+        Task<Accommodation> MergeData(RichAccommodationDetails accommodation);
```
[the remark this answers is not in the document]
Comment (hasmikharutyunyan) [reply HappyTravel.PropertyManagement.Api/Services/Workers/IAccommodationsDataMerger.cs:12]: First for one accommodation (it is understandable by parameter).
Second for all accommodations

**Comment by hasmikharutyunyan** [comment]
> Should't we rename the project to `AccommodationMapper`, what do you think?

I think it can be another name, because as I understand this service will be work and with predictions,
maybe `StaticDataManager`?

**Comment by kirillta** [comment]
Maybe, maybe

File: HappyTravel.PropertyManagement.Api/Services/Workers/IAccommodationsDataMerger.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (kirillta) [reply HappyTravel.PropertyManagement.Api/Services/Workers/IAccommodationsDataMerger.cs:12]: Please rename them

**Review by kirillta — APPROVED** [review]

**Pull request merged**